skillmap/src/lib/skillMapParser.ts

```diff
@@ -255,18 +255,21 @@ function inflateMapLayout(section: MarkdownSection, base: Partial<MapLayoutNode>
     return result as MapLayoutNode;
 }
 
-function inflateMapReward(section: MarkdownSection, base: Partial<MapReward>): MapReward {
-    const result: Partial<MapReward> = {
+function inflateMapReward(section: MarkdownSection, base: Partial<MapRewardNode>): MapRewardNode {
+    const result: Partial<MapRewardNode> = {
         ...base,
         kind: (section.attributes.kind || "reward") as any,
-        url: section.attributes.url
     };
 
     if (section.attributes["type"]) {
         const type = section.attributes["type"].toLowerCase();
         switch (type) {
             case "certificate":
-                result.type = type;
+                if (!result.rewards) result.rewards = [];
```

Comment:
for later: just noting that we should probably update the docs/sample repo with the new formatting once this is out so that we (hopefully) don't forget